src/networkProviders/networkConfig.ts

```diff
@@ -12,54 +11,55 @@ export class NetworkConfig {
     /**
      * The gas required by the Network to process a byte of the transaction data.
      */
-    public GasPerDataByte: number;
+    public GasPerDataByte: bigint;
```

Comment:
These use a different casing than other fields. Maybe rename them for consistency (even if that leads to a small breaking change)?